### PR TITLE
Conditionalize gpcc 

### PR DESCRIPTION
**Pull request by litzell**
distinguish between gpperfmon and Command Center and conditionalize gpcc.

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/intro/about_utilities.xml [hunk 1]
```diff
@@ -21,21 +21,26 @@
       <li>Transferring data between Greenplum databases</li>
       <li>System state reporting</li>
     </ul>
+    <p>Greenplum Database includes an optional performance management database that contains query
+      status information and system metrics. The <codeph>gpperfmon_install</codeph> management
+      utility creates the database, named <codeph>gpperfmon</codeph>, and enables data collection
+      agents that execute on the Greenplum master and segment hosts. Data collection agents on the
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/intro/about_utilities.xml]: Greenplum master -> Greenplum Database master

File: gpdb-doc/dita/admin_guide/managing/monitor.xml [hunk 2]
```diff
@@ -16,20 +16,21 @@
...
-          <codeph>gpperfmon</codeph> database and manages the data collection agents on the segment
-        hosts. Administrators can query metrics in the <codeph>gpperfmon</codeph> database.</p>
-      <p>The Greenplum Command Center Console is a web-based interface that graphically displays
-        metrics collected in the <codeph>gpperfmon</codeph> database and provides additional system
-        management tools. Greenplum Command Center ships separately from the Greenplum Database
-        distribution. Download the Greenplum Command Center package from <xref
-          href="https://network.pivotal.io" scope="external" format="html">Pivotal Network</xref>
-        and view documentation at the <xref href="http://gpcc.docs.pivotal.io" scope="external"
-          format="html">Greenplum Command Center Documentation</xref> web site. </p>
+      <p>Pivotal provides an optional system monitoring and management database,
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/monitor.xml]: Seems like this para shouldn't mention Pivotal and shouldn't be in a topic that's conditionalized for pivotal.  Isn't gpperfmon the shared db available in both?

File: gpdb-doc/dita/admin_guide/managing/monitor.xml [hunk 3]
```diff
@@ -16,20 +16,21 @@
...
+          <codeph>gpperfmon</codeph>, which administrators can enable with Greenplum Database. The
+          <codeph>gpperfmon_install</codeph> command-line utility creates the
+          <codeph>gpperfmon</codeph> database and enables data collection agents that collect and
+        store system metrics in the database. Administrators can query metrics in the
+          <codeph>gpperfmon</codeph> database. See the documentation for the
+          <codeph>gpperfmon</codeph> database in the <cite>Greenplum Database Reference
+        Guide</cite>.</p>
+      <p otherprops="pivotal">Pivotal Greenplum Command Center, an optional web-based interface,
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/monitor.xml]: This is already enclosed in an otherprops="pivotal" topic so doesn't strictly need the conditionalization here.

File: gpdb-doc/dita/admin_guide/managing/monitor.xml (hunk 2 above)
Comment (litzell) [inline gpdb-doc/dita/admin_guide/managing/monitor.xml]: Removed the otherprops attr from the topic. I intended the first paragraph to be unconditional and the 2nd to be conditional. Also replaced Pivotal with Greenplum Database.

**Pull request merged**